src/Shared/FileMatcher.cs

```diff
@@ -2275,6 +2301,51 @@ private static int SkipSlashes(string aString, int startingIndex)
             return index;
         }
 
+        /// <summary>
+        /// Returns true if drive enumerating wildcard patterns are detected using the directory and wildcard parts.
+        /// </summary>
+        private static bool IsDriveEnumeratingWildcardPattern(string directoryPart, string wildcardPart)
+        {
+            string combinedPart = Path.Combine(directoryPart, wildcardPart);
+            int combinedPartLength = combinedPart.Length;
+            if (combinedPartLength >= 7 &&                                                                  // Handles <drive letter>:\\** case
+                IsDrivePatternWithoutSlash(combinedPart[0], combinedPart[1]) &&
+                FileUtilities.IsAnySlash(combinedPart[2]) &&
```

Comment:
Shouldn't you be able to handle any number of slashes before the **?

Also, I think ladipro implemented something like this. Can you use his code or is it too far away?

Reply:
I think his code might be too far away - I reviewed his changes and tried a slightly different way to handle any number of slashes. Is there a better way to achieve this?